test/runtests.jl

```diff
@@ -171,22 +146,32 @@ function test_tracer_diagnostics(model)
     return nothing
 end
 
+
+scalar_diff = ScalarDiffusivity(ν=1e-6, κ=1e-7)
+
 @testset "Oceanostics" begin
     model_kwargs = (buoyancy = Buoyancy(model=BuoyancyTracer()), 
                     coriolis = FPlane(1e-4),
                     tracers = :b)
 
-    model = NonhydrostaticModel(; grid, model_kwargs...,
-                                closure = ScalarDiffusivity(ν=1e-6, κ=1e-7))
+    models = (NonhydrostaticModel(; grid, model_kwargs...,
+                                  closure = scalar_diff),
+              HydrostaticFreeSurfaceModel(; grid, model_kwargs...,
+                                          closure = scalar_diff))
 
-    @info "Testing velocity-only diagnostics"
-    test_vel_only_diagnostics(model)
+    for model in models
+        model_type = split(summary(model), "{")[1]
+        @info "Testing velocity-only diagnostics in $model_type"
+        #test_vel_only_diagnostics(model)
```

Comment:
Do you want these all commented out? Or was this just for your own testing of each element...

Reply:
Ah, thanks for catching that. Forgot to uncomment these lines :)